Utils/AppLogger.swift

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import XCGLogger
+
+public extension XCGLogger {
+    class func appInstance() -> XCGLogger {
+        struct statics {
+            static let instance: XCGLogger = {
+                var logger = XCGLogger.defaultInstance()
+                let cacheDir = NSSearchPathForDirectoriesInDomains(.CachesDirectory, .UserDomainMask, true).first as! String
+                let fileDestination = XCGFileLogDestination(owner: logger, writeToFile: "\(cacheDir)/app.log", identifier: XCGLogger.constants.baseFileLogDestinationIdentifier)
```

Comment:
What do you think about creating a new log file each cold app start identified by timestamp and we set a cap to the size limit of log files (like 5 mbs) and delete the older logs?